doc/fileformat/neuroml.rst

```diff
@@ -8,31 +8,32 @@ Arbor offers limited support for models described in
 This is not built by default, but can be enabled by
 providing the `-DARB_NEUROML=ON` argument to CMake at
 configuration time (see :ref:`install-neuroml`). This will
-build the ``arborio`` libray with neuroml support.
+build the ``arbornml`` libray and defines the corresponding
```

Comment:
I guess there was a merge error because we've reverted to `arbornml` instead of `arborio`